File: push_files.py
import json

files = [
    "Projeler/WhatsApp_Onboarding/config/env.js",
    "Projeler/WhatsApp_Onboarding/server.js",
    "Projeler/WhatsApp_Onboarding/services/manychat.js",
    "Projeler/WhatsApp_Onboarding/services/phoneValidator.js",
    "_knowledge/bekleyen-gorevler.md",
    "_knowledge/calisma-kurallari.md"
]

out = []
for f in files:
    try:
        with open(f, 'r', encoding='utf-8') as file:
            out.append({
                "path": f,
                "content": file.read()
            })
    except Exception as e:
        print(f"Error reading {f}: {e}")

with open("push_files.json", "w", encoding="utf-8") as f:
    json.dump(out, f)

File: push_files.js
const fs = require('fs');

const files = [
    "Projeler/WhatsApp_Onboarding/config/env.js",
    "Projeler/WhatsApp_Onboarding/server.js",
    "Projeler/WhatsApp_Onboarding/services/manychat.js",
    "Projeler/WhatsApp_Onboarding/services/phoneValidator.js",
    "Projeler/WhatsApp_Onboarding/package.json",
    "_knowledge/bekleyen-gorevler.md",
    "_knowledge/calisma-kurallari.md"
];

const out = [];
for (const f of files) {
    try {
        const content = fs.readFileSync(f, 'utf8');
        out.push({
            path: f,
            content: content
        });
    } catch (e) {
        console.error(`Error reading ${f}: ${e.message}`);
    }
}

fs.writeFileSync("push_files.json", JSON.stringify(out));
